docs/Java_API.md

```diff
@@ -1,10 +1,37 @@
 # ONNX Runtime Java API
 The ONNX runtime provides a Java binding for running inference on ONNX models on a JVM, using Java 8 or newer.
 
-Two jar files are created during the build process, one contains the onnxruntime shared library, the JNI binding and the Java class files, and the other only contains the class files. By default the shared libraries are loaded from the classpath in a folder called `/lib`, if you wish to have them load from `java.library.path` then supply `-DORT_LOAD_FROM_LIBRARY_PATH` to the JVM at runtime.
+Release artifacts are published to Maven Central for use as a dependency in most Java build tools. The artifacts are built with support for some popular plaforms.
+
+![Version Shield](https://img.shields.io/maven-central/v/com.microsoft.onnxruntime/onnxruntime)
+
+| Artifact  | Description | Supported Platforms |
+|-----------|-------------|---------------------|
+| [com.microsoft.onnxruntime:onnxruntime](https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime) | CPU | Windows x64, Linux x64, macOS x64 |
+| [com.microsoft.onnxruntime:onnxruntime_gpu](https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime_gpu) | GPU (CUDA) | Windows x64, Linux x64 |
+
+For building locally, please see the [Java API development documentation](../java/README.md) for more details.
+
+For customization of the loading mechanism of the shared library, please see [advanced loading instructions](../java/README.md#advanced-loading).
+
+## API Reference
+
+The Javadoc is available [here](https://javadoc.io/doc/com.microsoft.onnxruntime/onnxruntime).
 
 ## Sample Code
 
+An example implementation is located in
+[src/test/java/sample/ScoreMNIST.java](./src/test/java/sample/ScoreMNIST.java).
```

Comment:
This link does not work it should be "../java/src/test/java/sample/ScoreMNIST.java" instead